### ISSUE TITLE
symbol not found in flat namespace '_ffi_type_double' on M1 when running tests with tox

### ISSUE THREAD

**Issue by Ignalion** [certbot/certbot#9269]
If you're having trouble using Certbot and aren't sure you've found a bug or
request for a new feature, please first try asking for help at
https://community.letsencrypt.org/. There is a much larger community there of
people familiar with the project who will be able to more quickly answer your
questions.

## My operating system is (include version):
MacOS 12 Monterey (M1 chip)

## I installed Certbot with (snap, OS package manager, pip, certbot-auto, etc):
git clone

## I ran this command and it produced this output:
tox -e py3-cover

```
    import collections
    import threading
    import types
    import typing

    import cryptography
    from cryptography import utils
    from cryptography.exceptions import InternalError
>   from cryptography.hazmat.bindings._openssl import ffi, lib
E   ImportError: dlopen(/Users/ignalion/projects/certbot/.tox/py3-cover/lib/python3.8/site-packages/_cffi_backend.cpython-38-darwin.so, 0x0002): symbol not found in flat namespace '_ffi_type_double'
```

**Comment by Ignalion** [comment]
Well, issue was resolved with an upgrade to python 3.10.

**Comment by alexzorin** [comment]
Glad to see you got it working. 

I think those of us that develop Certbot on M1 Macs, we use Python from Homebrew, rather than the version of Python that comes with the system. I think that suggestion is softly made in https://eff-certbot.readthedocs.io/en/stable/contributing.html#running-a-local-copy-of-the-client.

I'm going to close this issue as I don't think there's anything we can change and your problem is resolved, but feel free to follow up with further questions.
